src/markdown/downloads/disbursements-by-month.md

```diff
@@ -24,34 +24,91 @@ Download fiscal year data:
 
 ## Data publication
 
 We publish the monthly file when we receive the data for the most recently available month, which is usually 1–2 months prior to the current month.
 
 ## Data dictionary
 
-Laws treat revenues from offshore natural resources and onshore natural resources differently. There are set percentages and amounts from each that go certain funds or local governments.
+### Fields and definitions
+Laws treat revenues from offshore natural resources and onshore natural resources differently. There are set percentages and amounts from each that go to specific funds, local governments, or government agencies. All revenue collected from Native American lands is disbursed back to Native American tribes, nations, or individuals.
 
 _Month_ The month the disbursement occurred
 
-_Year_ The year the disbursement occurred
+_Calendar Year_ The period between January 1 and December 31 for a given year.
 
 _Fund Type_ The type of recipient receiving the disbursement
 
+* _BLM-Nat Petrol Reserve-2_ Funds from oil and gas leases directed to the National Petroleum Reserve in Alaska.
+* _Bureau of Land Management_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Bureau of Ocean Energy Management_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Bureau of Safety & Environmental Enforcement_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Fish & Wildlife_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Forest Service_ Certain funds are directed back to the federal agencies that administer these lands and waters to help cover the agencies’ operational costs.
+* _Land & Water Conservation Fund_ This fund provides matching grants to states and local governments to buy and develop public outdoor recreation areas across the 50 states. The Gulf of Mexico Energy Security Act (GOMESA) of 2006 specifies that 12.5% of revenues from certain gulf leases be directed to the LWCF stateside program. States can receive up to $125 million a year in funding. This portion of LWCF funds is not subject to the congressional appropriation process, but are otherwise treated similarly to regular LWCF funds.
+* _Lease Process Improvement (BLM)_ Permit Processing Fund, Bureau of Land Management (BLM), was established by the Department of the Interior as authorized by 119 Stats. 725, 671, and 695, for the deposit of mineral lease rentals. The Department of the Interior is authorized to use funds for the coordination and processing of oil and gas use authorizations on Federal land onshore, under the jurisdiction of the Pilot Project offices.
```

Comment:
This line is complex and difficult to read. Is there a way to simplify this, especially the statutes part?